Description: removed the forced -std=c++98
 GDCM includes headers that require at least -std=c++11
Author: Gert Wollny <gewo@debian.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,7 +15,6 @@
   cmake_policy(SET CMP0063 NEW)
 endif()
 project(GDCM)
-set(CMAKE_CXX_STANDARD 98)
 set(CMAKE_CXX_EXTENSIONS OFF)
 mark_as_advanced(CMAKE_BACKWARDS_COMPATIBILITY CMAKE_BUILD_TYPE CMAKE_INSTALL_PREFIX)
 set(GDCM_CMAKE_DIR "${GDCM_SOURCE_DIR}/CMake" CACHE INTERNAL "")
